### ISSUE TITLE
Error summary list not read out in Safari + Voiceover

### ISSUE THREAD

**Issue by vanitabarrett** [alphagov/govuk-frontend#2072]
Original issue submitted by Chris Moore via Zendesk: https://govuk.zendesk.com/agent/tickets/4315309

## Description of the issue
List of error messages within the error summary is not announced when using Voiceover and Safari (all other browser/screen-reader combos are OK)

## Steps to reproduce the issue
On this page, submit the form without filling in any fields. An error summary appears, but the error list is not read out.
https://www.immigration-health-surcharge-reimbursement.service.gov.uk/claim/name

<img width="1440" alt="Screenshot 2020-11-10 at 14 12 09" src="https://user-images.githubusercontent.com/29889908/102228666-dc086200-3ee2-11eb-9418-d6a64fa03131.png">

## Actual vs expected behaviour
When focus lands on the error summary, it should read out the heading and then the list of errors automatically. This happens correctly on the blue badge service, but for ours it only reads the heading.

I found the following issue in GitHub - https://github.com/alphagov/govuk-frontend/issues/1471 - but the workaround of adding role=“list” to the <ul> made no difference.

## Environment (where applicable)

- Operating system: macOS Catalina (Version 10.15.5)
- Browser: Safari (Version 13.1.1)

**Comment by tvararu** [comment]
I tested the link on VoiceOver + Safari on an iPad Pro (10.5") running iOS 15.4.1, and I can't reproduce the issue.

Here is a screenshot, with the captions at the bottom:

![IMG_0220](https://user-images.githubusercontent.com/1650875/172138688-35c5fa7f-781e-4747-a4d4-a24d38bdf70f.PNG)

Haven't tested on macOS but my assumption is it should also work (VoiceOver tends to behave similarly but not always)
